src/SDK/SmartDetectors.Extensions/AlertExtensions.cs

```diff
@@ -270,10 +270,14 @@ private static AlertProperty CreateAlertProperty(Alert alert, PropertyInfo prope
                             columnProperty.GetValue(tableRows[i]));
                     }
 
-                    columns.Add(new TableColumn(columnProperty.Name, tableColumnAttribute.DisplayName));
+                    // Store the column and its order
+                    columnsWithOrder.Add(Tuple.Create(new TableColumn(columnProperty.Name, tableColumnAttribute.DisplayName), tableColumnAttribute.Order));
                 }
             }
 
+            // Get the columns in order
+            List<TableColumn> columns = columnsWithOrder.OrderBy(t => t.Item2).Select(t => t.Item1).ToList();
```

Comment:
>t => t.Item2 [](start = 65, length = 12)

what would happen if i don't put any order?

Reply:
By default, Order is byte.MaxValue (same as any alert property)

---
In reply to: [254183285](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/46#discussion_r254183285) [](ancestors = 254183285)